config/settings.yml

```diff
@@ -1135,12 +1135,13 @@
       :chargeback_generation_interval: 1.day
       :chargeback_generation_time_utc: 01:00:00
       :compliance_purge_interval: 1.day
+      :container_scan_dispatcher_interval: 15.seconds
       :db_diagnostics_interval: 30.minutes
       :drift_state_purge_interval: 1.day
       :event_streams_purge_interval: 1.day
       :evm_snapshot_delete_delay_for_job_not_found: 1.hour
       :evm_snapshot_interval: 1.hour
-      :job_proxy_dispatcher_interval: 15.seconds
```

Comment:
I assume we'll need a data migration for this?

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/591